### ISSUE TITLE
Make minimap data-driven

### ISSUE THREAD

**Issue by kelsey-taylor** [earthrise-media/vacs-map#89]

**Comment by mwbernard** [comment]
@kelsey-taylor missing data for Maize, Tomato, and Soybean - but otherwise this looks good. I guess we'll just use the fallback color (none) for those?

**Comment by kelsey-taylor** [comment]
@mwbernard yep that works! i knew some of them were missing but bummer i guess that it's three of the biggies 🤷‍♀️
